.github/workflows/packager.yml

```diff
@@ -0,0 +1,318 @@
+name: packager
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - '**'
+    tags:
+      - 'v*.*.*'
+      - 'v*.*.*-*'
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@master
+        with:
+          go-version: 1.19
+      - name: Generate release tag
+        id: tag
+        run: echo "::set-output name=release_tag::bor_$(date +"%Y.%m.%d_%H-%M")"
```

Comment:
The release name should match the release tag exactly (no date/time info should be included), as what we've been doing in the past: https://github.com/maticnetwork/bor/releases/

Reply:
Updated and now using tag